tests/test_inequality.py

```diff
@@ -81,3 +82,61 @@ def test_theil_1d_1_w():
     obtained = inequality.theil(income=x, weights=w)
     expected = 0
     assert obtained==expected
+
+def test_ratio_eqaulity():
+    x = np.array([1, 9])
+    w = np.array([9, 1])
+    obtained = inequality.ratio_top_rest(income=x, weights=w)
+    assert obtained == 1.0
+
+def test_ratio_equality_fracc():
+    x = np.array([1, 9])
+    w = np.array([.9, .1])
+    obtained = inequality.ratio_top_rest(income=x, weights=w)
+    assert obtained == 1.0
+
+def test_ratio_1d():
+    x = np.array([57, 63, 81, 79, 88, 57, 42, 3, 77, 89])
```

Comment:
Interesting, this is a special case that I never think about where we have the same class `57` twice with a weight of 2 and 7... 

I think they should be merged into 9. I will think about it further.

Reply:
Oh, I just generated a random array with numpy, and then I calculated the value using a spreadsheet.

Even if it works in this case, the test shouldn't check it. I'll change it so `57` only appears one time (with a weight of 9)